build_daemon/test/daemon_test.dart

```diff
@@ -39,6 +36,18 @@ void main() {
       }
     });
 
+    test('can be stopped', () async {
+      var workspace = uuid.v1();
+      testWorkspaces.add(workspace);
+      var daemon = Daemon('$workspace');
+      expect(daemon.tryGetLock(), isTrue);
+      var timeout = Duration(seconds: 30);
```

Comment:
30 seconds seems like a lot, if it waits that long to actually shut down I would reduce this to make the test run faster (5 seconds?)

Reply:
It shuts down immediately. This is just in case the shutdown logic doesn't work which actually should be handled by the test infra so I'm gonna remove it.

Reply:
kk